lib/ood_core/job/adapters/torque.rb

```diff
@@ -85,9 +85,9 @@ def submit(script, after: [], afterok: [], afternotok: [], afterany: [])
           depend << "afterany:#{afterany.join(':')}"     unless afterany.empty?
 
           # Set mailing options
-          mail_points  = ""
-          mail_points += "b" if script.email_on_started
-          mail_points += "e" if script.email_on_terminated
+          mail_points = ""
+          mail_points.concat "b" if script.email_on_started
+          mail_points.concat "e" if script.email_on_terminated
```

Comment:
In this PR you need to cherry-pick https://github.com/OSC/ood_core/pull/192/commits/f68fc94f9172783c753e1ef7a925a88ab09a36e4 which was lost when you re-did.